prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -101,10 +101,14 @@ EVE_OPT=" --jsons-folder $EDJSONS_DIR"
 
 [[ "0$DISABLE_ROOT_OUTPUT" == "00" ]] && DISABLE_ROOT_OUTPUT=
 
+has_detectors_gpu TPC ITS && GPU_INPUT+=",its-clusters"
+has_detectors_gpu TPC ITS && GPU_OUTPUT+=",its-tracks"
+
 if [[ $CTFINPUT != 1 ]]; then
   GPU_OUTPUT+=",tpc-triggers"
 fi
 if [[ $SYNCMODE == 1 ]]; then
+  has_detectors_gpu TPC ITS && ITS_CONFIG_KEY+="ITSCATrackerParam.trackingMode=0;" # sets ITS gpu reco to sync
```

Comment:
@davidrohr it is not supposed to have ITS gpu in sync configuration. But there are here two choices: either we don't set the reco type for ITS and let it crash or we make it consistent and set sync mode.
The crash message would be meaningful, but in principle, as for us is just matter of configuration, I'd go for a functioning version that runs also in sync.

Let me know, if you don't like it I can remove this line.

Reply:
This line is correct, since by default in sync reco at P2 we won't have WORKFLOW_DETECTORS_GPU=ITS, so it will not be effective. While for local tests with ITS on GPU it will be, so this is fully correct.